yaml/Codat-Expenses.yaml

```diff
@@ -984,43 +991,47 @@ components:
       type: string
       description: Status of the transaction.
       enum:
-        - Unknown
-        - Pending
-        - ValidationError
-        - Completed
-        - PushError
+      - Unknown
```

Comment:
**Error 1**: validate-enums - Enum is nullable but does not contain a null value

**Suggestion 1**: Add null to the enum values of TransactionStatus

*Applied around line 0*
**Suggestion 1**: **Suggestion 1**: Add null to the enum values of TransactionStatus

**Explanation 1**: The error message indicates that there is an enum that is nullable but does not contain a null value. The enum in question is likely the one defined under TransactionStatus. The enum is indeed nullable, as indicated by nullable: true, but the enum values do not include null. Adding null to the enum values should resolve the error.